src/manifold/src/boolean_result.cpp

```diff
@@ -255,7 +255,7 @@ void AddNewEdgeVerts(
         [&](size_t hash) { mutexes[hash % mutexes.size()].unlock(); },
         std::placeholders::_1);
     tbb::parallel_for(
-        tbb::blocked_range<size_t>(0_z, p1q2.size(), 32),
+        tbb::blocked_range<size_t>(0_uz, p1q2.size(), 32),
```

Comment:
FWIW, `tbb::blocked_range` will inver its type, so `tbb::blocked_range(0_uz, p1q2.size(), 32),` should work just as well.

Reply:
👍 good fodder for a follow-on